module-2/backend/week_one.md

```diff
@@ -5,30 +5,125 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  POST, GET, PUT, DELETE
+
 2. What is Sinatra?
+  A lightweight version of Ruby on Rails.

```

Comment:
sinatra and rails are complete different web application frameworks. they both are written in Ruby, but they are each their own Domain Specific Language